compiler/z/codegen/OMRTreeEvaluator.hpp

```diff
@@ -235,13 +235,21 @@ class OMR_EXTENSIBLE TreeEvaluator: public OMR::TreeEvaluator
     *
     *   \param cg
     *      The code generator used to generate the instructions
+    * 
+    *   \param shiftAmount
+    *      If using function to replace a shift+and combination, this is the variable that holds how much we want to shift.
+    *      If we are using this function to replace a standalone 'and', then this variable should be zero.
```

Comment:
The documentation for this parameter is incomplete. The parameter type is signed on purpose but it does not tell the user of the API what it means for a value to be positive vs. negative. How about:

```
    *   \param shiftAmount
    *      The number of bits we should shift the result of the logical AND. A positive value represents a left shift,
    *      a negative value represents a right shift, and a value of zero represents no shift.
```